src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
-
+enum Shape {
+  Triangle,
+  Circle,
+  Rectangle
 }
 
-export class Triangle {
+enum Color {
+  Red,
+  Green,
+  Blue
+}
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = Shape.Triangle;
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Enter correct values of triangle sides');
+    }
+  }
+
+  getArea(): number {
+    const sp = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(sp * (sp - this.a) * (sp - this.b) * (sp - this.c));
 
+    return area;
```

Comment:
a method getArea returns the area of the figure _**rounded down to hundredths**_